codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/generators/StructureGenerator.kt

```diff
@@ -65,17 +72,10 @@ class StructureGenerator(
     }
     private val name = symbolProvider.toSymbol(shape).name
 
-    fun render() {
-        renderStructure()
-        errorTrait?.also { errorTrait ->
-            ErrorGenerator(symbolProvider, writer, shape, errorTrait).render()
-        }
-    }
-
-    fun renderServer() {
+    fun render(forWhom: TYPE = TYPE.CLIENT) {
```

Comment:
Do you really need to specify that `fromWom` is a `TYPE` ? Doesn't the compiler guess it by itself?

Reply:
The fact that `fromWhom` is of type `TYPE`. I would expect this to also work: `fun render(fromWhom = TYPE.CLIENT)` but I am probably wrong.